Library/Formula/node.rb

```diff
@@ -14,6 +14,7 @@ class Node < Formula
   end
 
   option "with-debug", "Build with debugger hooks"
+  option "with-icu4c", "Build with Intl (icu4c) support"
```

Comment:
Or simply remove this as an option altogether, since I don't see much reason why one should wan't to **not** link to the system lib, as it's perfectly adequate?